exercises/rectangles/rectangles.py

```diff
@@ -1,2 +1,2 @@
-def count():
+def count(string):
```

Comment:
I think `ascii_diagram` would be clearer. Do you agree?

Reply:
yup. changed!